flake-info/src/data/export.rs

```diff
@@ -224,6 +223,23 @@ impl TryFrom<import::NixpkgsEntry> for Derivation {
                     .map(|l: &License| l.fullName.to_owned())
                     .collect();
 
+                let platforms: HashSet<String> = package
+                    .meta
+                    .platforms
+                    .map_or(Default::default(), Flatten::flatten)
+                    .into_iter()
+                    .collect();
+
+                let bad_platforms: HashSet<String> = package
+                    .meta
+                    .bad_platforms
+                    .map_or(Default::default(), Flatten::flatten)
+                    .into_iter()
+                    .collect();
+
+                let package_platforms: Vec<String> =
```

Comment:
```suggestion
                let platforms: Vec<String> =
```

Not a strong opinion, but reassigning platforms (or assigning it in a scope) would avoid using the wrong set later on